sysutils/screen/Portfile

```diff
@@ -53,7 +53,7 @@ post-extract {
 }
 
 use_autoconf    yes
-autoconf.cmd    ./autogen.sh
+autoconf.cmd    autoreconf --install
```

Comment:
it look like from the [guide](https://guide.macports.org/#reference.phases.configure.automake-autoconf) that you should just set `use_autoreconf  yes` and remove the two "autoconf" lines

Reply:
Good catch